tests/publiccloud/create_aistack_env.pm

```diff
@@ -219,30 +272,27 @@ sub install_nvidia_drivers {
 
     record_info('Install nvidia drivers');
     script_run("curl -sSL $easyinstall_url -o ./nvidia_easy_install.sh && chmod +x ./nvidia_easy_install.sh");
```

Comment:
Here you download and prepare `nvidia_easy_install.sh` script, but execution has been **commented[cancelled]**: still this `curl` is needed?.

If answer is YES, kinldy explain why does and please change to `assert_...` form, also in below `zypper ar ...`